config/deploy/staging.rb

```diff
@@ -7,7 +7,7 @@
 # server "example.com", user: "deploy", roles: %w{app web}, other_property: :other_value
 # server "db.example.com", user: "deploy", roles: %w{db}
 
-server "restarone-solutions", user: 'ubuntu', roles: %w{app db web}
+server "54.163.245.12", user: 'ubuntu', roles: %w{app db web}
```

Comment:
we should pass this value dynamically + allow the production deployment to be passed an IP address as well